### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.13 esercizio 1 (4 dicembre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#336] [Esercizi M-Z]
Sia $X$ v.a. $\text{unif} \big( (-3,10) \big)$. Calcolare:
- $E[X]$
- $E[2X^{3}]$

Poi calcolare $E[X]$ con $X$ definita come nel 2° esercizio di lunedi 02 dicembre (vedi #335)

**Comment by rimaout** [comment]
<img width="810" alt="image" src="https://github.com/user-attachments/assets/9af467b4-413e-4c43-bd58-3525611e4d0a">

**Comment by alem1105** [comment]
<img width="762" alt="image" src="https://github.com/user-attachments/assets/f483fd45-c1c9-499f-9148-57c5f98ac0df">
